tests/test_json_schema.py

```diff
@@ -4264,8 +4264,8 @@ class Foobar(BaseModel):
 
 def test_override_generate_json_schema():
     class MyGenerateJsonSchema(GenerateJsonSchema):
-        def generate(self, schema, mode='validation'):
-            json_schema = super().generate(schema, mode=mode)
+        def generate(self, schema, mode='validation', sort_schema=True):
+            json_schema = super().generate(schema, mode=mode, sort_schema=sort_schema)
```

Comment:
Could you add a section to this test that verifies the operation of the `sort_schema=False` flag in the case of a custom generate json schema function?

Reply:
do you mean parameterize the flag?

Reply:
Why was it closed? It really a very nice thing to have. Why would we need to sort fields in schema examples like {user..., password...} into {password..., user...}  - it looks awful. Pls reopen it guys

Reply:
See https://docs.pydantic.dev/latest/concepts/json_schema/#json-schema-sorting